.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,34 @@
+name: Bug report
+description: Create a report to help us improve
+labels: [bug, optimization, good first issue]
```

Comment:
I mean, it's something for us to decide. If you look at more or less big python and non-python projects out there, they all have this "good first issue" label. This helps new people, who are looking for an easier issue to tackle, to find one. No strong opinion, though, this might be a bit too early for it, we can simply vote :)

As for the optimization, on the second thought, that one might not be needed. Let me remove it for now. We can always adjust it later if need be.

Reply:
I'm voting for removing `optimization` but keeping `good first issue`

Reply:
Done